sceptre/plan/actions.py

```diff
@@ -1095,15 +1095,15 @@ def _log_drift_status(self, response: dict) -> None:
 
         for key in keys:
             if key in response:
-                self.logger.debug(
+                self.logger.info(
```

Comment:
I'd probably say keep this at debug level. Otherwise it's going to be REALLY chatty for a large stack group.

Reply:
ok